tests/trestle/tasks/xlsx_to_oscal_profile_test.py

```diff
@@ -79,6 +79,32 @@ def test_xlsx_execute(tmp_path):
         assert (result)
 
 
+@patch(target='uuid.uuid4', new=uuid_mock1)
+@patch(target='trestle.tasks.xlsx_to_oscal_profile.get_trestle_version', new=get_trestle_version_mock1)
+def test_xlsx_execute_filter(tmp_path):
+    """Test execute filter call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/xlsx/test-xlsx-to-oscal-profile.config')
+    config.read(config_path)
```

Comment:
Might be good to define a const path for tasks in test_utils.py.  See the top of that file

Reply:
Fixed, kinda. The file path reference is specific to this test script. Same for xlsx_to_oscal_cd_test. I'm not so sure moving to test_utils.py is the best option. However, I did make a constant variable for each at the respective tops.